### DISCUSSION TITLE
blocking outfit from install due to another outfit

### DISCUSSION THREAD

**Discussion by daggs1** [endless-sky/endless-sky#7655] [General]
Greetings,

given outfit A and outfit B, does the game has the ability to prevent install of outfit A if outfit B is installed and vice versa?

Thanks

**Comment by vitalchip** [comment]
Hi, 

There isn't really a way to do this as standard, you can use attributes for a very crude method of controlling interaction.

For example you might have two outfits and want to allow either x or y on a ship but not both, so you give the ship an attribute (either directly or via an outfit) for example "banana 1"  and then give each outfit the attribute "banana -1"  So when you install the first outfit it will reduce the ship's "banana attribute" to zero, this will stop the second being installed becausde the count isn't allowed to be negative.

It's a bit clumsy and it also means you have to be careful when uninstalling that it is done in the right order but (unless anything has changed within the last few months) it's what we have.

**Reply by daggs1** [reply]
thanks for the info
